macos/ReactTestApp/AppDelegate.swift

```diff
@@ -63,7 +73,9 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
             item.isEnabled = false
         }
 
+        // 0             1     2     3       4     5
         // ReactTestApp  File  Edit  Format  View  React  Window  Help
+        //                                         ^~~~~ Insert `reactMenu` here
```

Comment:
Similarly to my above comment, it might be more consistent to instantiate the menu in your nib/storyboard, like all the other existing menu items; _or_ create all of them programmatically, which seems like overkill–there may also be a way to insert an item by means of ‘menu validation’, but it’s been too long for me to remember off hand.